src/rubyfmt.rb

```diff
@@ -2401,8 +2209,251 @@ def format_redo(ps, expression)
   ps.emit_newline if ps.start_of_line.last
 end
 
+def format_splat(ps, rest)
+  ps.emit_ident("*")
+  format_expression(ps, rest[0])
+end
+
+def format_to_proc(ps, rest)
+  ps.emit_ident("&")
+  format_expression(ps, rest[0])
+end
+
+def format_keyword(ps, rest)
+  ps.emit_ident(rest[0])
+end
+
+def format_method_call(ps, rest)
+  ps.emit_indent if ps.start_of_line.last
+
+  chain, method, original_used_parens, args = rest
+
+  args = args.map { |a| normalize(a) }
```

Comment:
This is kind of a hack: the args would get normalised anyway when we pass them to `format_expression`, but by pre-normalising them we can check if any of them are a `:method_call` instead of having to check for all the different Ripper variants on method calls.